.github/workflows/release-wc-and-playground.yml

```diff
@@ -0,0 +1,71 @@
+name: Release Web-component and Playground
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+
+  webcomponent:
+    name: Release Web-component
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: Install latest version of library in the web-component
+        # sleep for 60 seconds before using latest version in web-component, because sometimes NPM needs additional few seconds to `save` package in registry
+        run: |
+          sleep 1m
```

Comment:
As I understand, this script will run parallel with `if-nodejs-release.yml` and for that we have `1m` sleep? Another question: What about bumping versions? As I understand from scripts, we will have 2 extra commits with bumps on master: first for react-component, then for playground and web-component, yes?

Reply:
Sleep was there already, we needed it no matter if it is in the release workflow or here in the new one.
They do not run in parallel. The release is first as it is triggered by the push to master. This workflow starts when the last step of the release finish (GitHub release creation). In theory, no sleep needed, but you remember we had this problem with "moment of publish to npm" vs "when it is actually available for install"

Reply:
So `if-nodejs-release.yml` and `release-wc-and-playground.yml` won't be triggered in this same time?

Reply:
tl;dr no 😄 

`if-nodejs-release.yml` is triggered on `push` to master, and `release-wc-and-playground.yml` on `release`. So the 2nd one will be triggered once [this](https://github.com/asyncapi/asyncapi-react/pull/218/files#diff-8ed11ec3da81556149f1f4a379d4229113f23d9ae8c4eea626132bac8d535ea6R53-R63) creates a release


--- 
while writing this comment I realized `release-wc-and-playground.yml` is now triggered on `push` and not `release` 😅 pushing fix in a minute

Reply:
Done 🥇